django/http/response.py

```diff
@@ -492,12 +492,15 @@ class JsonResponse(HttpResponse):
       ``django.core.serializers.json.DjangoJSONEncoder``.
     :param safe: Controls if only ``dict`` objects may be serialized. Defaults
       to ``True``.
+    :param json_dumps_params: A dictionary of kwargs that would be passed
+      to json.dumps
     """
 
-    def __init__(self, data, encoder=DjangoJSONEncoder, safe=True, **kwargs):
+    def __init__(self, data, encoder=DjangoJSONEncoder,
+                    json_dumps_params={}, safe=True, **kwargs):
```

Comment:
`json_dumps_params` should be after `safe` (reordering keywords could be backwards incompatible if passing them by argument).